build/notebooks/table_variants_missense.md

```diff
@@ -5409,7 +5588,7 @@ prologue = r"""
 \cmidrule(r){13-14}
 Position\tablefootnote{Position relative to AgamP3 reference sequence, chromosome arm 2L.} & 
 \emph{Ag}\tablefootnote{Codon numbering according to transcript \texttt{AGAP004707-RA} in geneset AgamP4.4.} & 
-\emph{Md}\tablefootnote{Codon numbering according to @@TODO.} & 
+\emph{Md}\tablefootnote{Codon numbering according to \emph{Musca Domestica Vgsc} EMBL accession X96668 (Williamson \emph{et al}., 1996).} & 
```

Comment:
Could you change "Domestica" to "domestica".

Also, could you change the Williamson citation to be a proper citation. This means using a latex ``\cite{}`` command, and adding the reference into ``refs.bib``. There is an example of how to do a citation in the main.tex intro section.